docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -0,0 +1,31 @@
+# Check Update Procedure
+
+## 1. Check Discovery Methods
+
+### 1.1 Consider Similar Tokens
+### 1.2 Java Enhancement Proposals (JEPS)
+### 1.3 Always Impacted Checks
+### 1.4 Real Usages Examples in Large codebases
```

Comment:
Let's make capitalization consistent, please use "title case" for headers, for all headers.